reference/release-notes/1.27.0.md

```diff
@@ -0,0 +1,78 @@
+
+
+### Other
+
+* Android security updates for August 2025 (see [Android Security Bulletin - August 2025](https://source.android.com/docs/security/bulletin/2025-08-01)).
+* The Android WebView has been updated to [138.0.7204.63](https://chromereleases.googleblog.com/2025/06/chrome-for-android-update_30.html).
+
+## Deprecations and removed functionality
+
+* Support for Juju 2.9 was deprecated in 1.25.1 and is removed with this release. Instead, upgrade to the latest Juju version, 3.6. See {ref}`sec-juju-version-requirements` for more information.
+* Support for kernels older than 6.8 was deprecated in 1.25.0 and is removed with this release.
+
+## Known issues
+
+* Due to a [bug in Juju](https://github.com/juju/juju/issues/20344), bootstrapping Juju on AWS with the latest Amazon Machine Image (AMI) fails. The AMI uses a Ubuntu 24.04 (Noble Numbat) based image by default. This issue can be temporarily resolved by forcing the bootstrap to use a Ubuntu 22.04 (Jammy Jellyfish) image.
+
+So when {ref}`deploying Anbox Cloud <sec-deploy-anbox-cloud-juju>`, use `--boostrap-base=ubuntu@22.04` with the `juju deploy` command:
+
+      juju deploy anbox-cloud-core --boostrap-base=ubuntu@22.04 --overlay ua.yaml
+      juju deploy anbox-cloud --boostrap-base=ubuntu@22.04 --overlay ua.yaml
+
+See our [open bugs in Launchpad](https://bugs.launchpad.net/anbox-cloud/?field.searchtext=&orderby=-importance&field.status%3Alist=NEW&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.status%3Alist=INPROGRESS&field.status%3Alist=DEFERRED&field.status%3Alist=FIXCOMMITTED&field.status%3Alist=INCOMPLETE_WITH_RESPONSE&field.status%3Alist=INCOMPLETE_WITHOUT_RESPONSE&assignee_option=any&field.assignee=&field.bug_reporter=&field.bug_commenter=&field.subscriber=&field.structural_subscriber=&field.milestone%3Alist=102837&field.tag=&field.tags_combinator=ANY&field.has_cve.used=&field.omit_dupes.used=&field.omit_dupes=on&field.affects_me.used=&field.has_patch.used=&field.has_branches.used=&field.has_branches=on&field.has_no_branches.used=&field.has_no_branches=on&field.has_blueprints.used=&field.has_blueprints=on&field.has_no_blueprints.used=&field.has_no_blueprints=on&search=Search) that are planned to be fixed for the 1.27.1 release, next month.
+
+## CVEs
+
+The Anbox Cloud 1.27.0 release includes fixes from the respective upstreams and components for the following CVEs:
+
+| CVE | Affected Components |
+|-----|---------------------|
+
+## Bug fixes
+
+* [LP 2110219](https://bugs.launchpad.net/anbox-cloud/+bug/2110219) When running an Anbox instance with an NVIDIA GPU and utilizing it for rendering and video encoding, the `lxcfs` process on the host shows an unusually high CPU consumption (~60% for a single Anbox instance).
+* [LP 2111598](https://bugs.launchpad.net/anbox-cloud/+bug/2111598) In an Android 15 instance with an NVIDIA GPU, when the developer settings are enabled, trying to access the *System* section of the *Settings* app causes an OS-level crash.
```

Comment:
This is not yet fixed (it's 'In Progress' on LP)

Reply:
Yes, this is not fixed yet.
We need to mention this as a known issue